### ISSUE TITLE
img_pipe + NWB

### ISSUE THREAD

**Issue by bendichter** [ChangLabUcsf/img_pipe#34]
Andrew has helped me build an extension for NWB to accommodate surface data. Any interest in adding it to this library as the canonical NWB extension for img_pipe?

**Comment by bendichter** [comment]
@libertyh

**Comment by libertyh** [comment]
Yes -- can you describe what exactly this would do/involve?  Does the NWB dataset include only imaging related data, or also electrophys data?

**Comment by bendichter** [comment]
The NWB format can support ephys data natively, however there was previously no structure that supports mesh data types. I worked with Andrew on the NWB dev team to create an extension to NWB that supports mesh data and would allow you to store our recons in NWB format. This is simply a text spec file that tells NWB about the structure of mesh data.

I think it would be better to separate ephys and anatomical data so that you don't have data duplication problem with every session's ephys file. Instead, hdf5 and nwb support soft links across files, so you could have a view of anatomical data within each session file without duplicating anatomical data.
